src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -444,11 +444,14 @@ private static bool AddNonCultureResources(Project project, List<string> compile
             return true;
         }
 
-        private static bool GenerateCultureResourceAssemblies(Project project, List<LibraryExport> dependencies, string outputPath)
+        private static bool GenerateCultureResourceAssemblies(
+            Project project,
+            List<LibraryExport> dependencies,
+            string intermediateOutputPath,
+            string outputPath)
         {
             var referencePaths = CompilerUtil.GetReferencePathsForCultureResgen(dependencies);
-            var resgenReferenceArgs = referencePaths.Select(referencePath => $"-r \"{referencePath}\"").ToList();
-
+            var resgenReferenceArgs = referencePaths.Select(path => $"-r:{path}").ToList();
```

Comment:
I looked at the code for Compiler.Csc which uses the response file format and they trim out quotes around the input. Unless I'm missing something, the quotes don't seem to do much here.